dev/docker/kerberos-hive/mapred-site.xml

```diff
@@ -0,0 +1,31 @@
+<configuration>
+  <property>
+    <name>mapreduce.framework.name</name>
+    <value>yarn</value>
```

Comment:
So here you configured yarn, that's why it must run on yarn. @mchades .

We don't have to set this, it will run locally, can you try this.

Reply:
Should we modify Docker without Kerberos by the way about this point? @jerryshao

Reply:
Yes, I think so. Besides, can you try if it works or not?

Reply:
> So here you configured yarn, that's why it must run on yarn. @mchades .
> 
> We don't have to set this, it will run locally, can you try this.

You are right, I have already verified it locally. We just need to change the value to `local`, and then we can execute HQL in the Hive server without relying on yarn.